src/Symfony/Component/DependencyInjection/Tests/Fixtures/xml/services9.xml

```diff
@@ -114,7 +117,7 @@
     </service>
     <service id="factory_simple" class="SimpleFactoryClass" public="false">
       <argument>foo</argument>
-      <deprecated>The "%service_id%" service is deprecated. You should stop using it, as it will be removed in the future.</deprecated>
+      <deprecated message="The &quot;%service_id%&quot; service is deprecated. You should stop using it, as it will be removed in the future."/>
```

Comment:
similar to yaml, we could use a new tag and provide a BC layer
`<deprecation package="..." version="..." id="...">Service %s is deprecated.</deprecation>`

Reply:
well, why a new tag ? We could add support for adding attributes on the existing tag, without it being a BC break.

Reply:
Oh: because I would suggest making package and version required.
But we could also keep the name and trigger a deprecation if that's what we prefer.

Reply:
I suggest triggering a deprecation (you would have to do it anyway when the old tag is used...)

It means we would have to wait until 6.0 to mark the attributes as required in the XSD, but that's fine IMO.

Reply:
@stof @nicolas-grekas if i understand well, we'll use the current tag (with new attributes) and deprecate not setting them ?